### PR TITLE
Upstream get-animations.html from Blink

### PR DESCRIPTION
**Pull request by alancutter**
Import https://chromium.googlesource.com/chromium/src/+/813dc6d714d81f658e24afde116b0e8f673eb660/third_party/WebKit/LayoutTests/web-animations-api/w3c/get-animations.html

### CODE REVIEW

File: web-animations/interfaces/Animatable/getAnimations.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Animatable.getAnimations tests</title>
+<link rel="help" href="https://w3c.github.io/web-animations/#dom-document-getanimations">
```
Comment (suzyh) [inline web-animations/interfaces/Animatable/getAnimations.html]: Should this be https://w3c.github.io/web-animations/#dom-animatable-getanimations ?

File: web-animations/interfaces/Animatable/getAnimations.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}, 'Test getAnimations on separate elements with separate animations');
+
+test(function(t) {
+  var divParent = createDiv(t);
+  var divChild = createDiv(t);
+  divParent.appendChild(divChild);
+  var animation = divChild.animate(null, 100 * MS_PER_SEC);
+  assert_array_equals(divParent.getAnimations(), [], 'divParent');
+  assert_array_equals(divChild.getAnimations(), [animation], 'divChild');
+}, 'Test getAnimations on element with animation on child element');
```
Comment (suzyh) [inline web-animations/interfaces/Animatable/getAnimations.html]: If we're adding this one I think it's worth having the reverse as well, or having animA on divParent and animB on divChild.

File: web-animations/interfaces/Animatable/getAnimations.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+test(function(t) {
+  var div = createDiv(t);
+  var animation = div.animate(null, 100 * MS_PER_SEC);
+  animation.finish();
+  assert_array_equals(div.getAnimations(), []);
+}, 'Test getAnimations on element with finished fill none animation');
```
Comment (suzyh) [inline web-animations/interfaces/Animatable/getAnimations.html:48]: It looks like the Blink test file also tests the case where the animation hasn't started having an effect yet (by having a delay set). Add that in too?

File: web-animations/interfaces/Animatable/getAnimations.html (hunk 2 above)
Comment (alancutter) [inline web-animations/interfaces/Animatable/getAnimations.html]: Done.

File: web-animations/interfaces/Animatable/getAnimations.html (hunk 1 above)
Comment (alancutter) [inline web-animations/interfaces/Animatable/getAnimations.html]: Good catch, done.

File: web-animations/interfaces/Animatable/getAnimations.html (hunk 3 above)
Comment (alancutter) [inline web-animations/interfaces/Animatable/getAnimations.html:48]: Done.

**Comment by suzyh** [comment]
lgtm

**Pull request merged**